kount/util/payment.py

```diff
@@ -15,17 +18,18 @@
 class Payment(object):
```

Comment:
More payment types are required, check https://api.test.kount.net/rpc/support/payments.html for full list.